client/package.json

```diff
@@ -18,6 +18,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --coverage --watchAll=false",
+    "testWatch": "react-scripts test --watchAll=true",
```

Comment:
Just a reminder to add a corresponding README entry if this is something we're going to keep.

Reply:
done